src/main/java/wtf/g4s8/examples/spaxos/InMemoryAcceptor.java

```diff
@@ -76,4 +79,9 @@ public synchronized void accept(final Proposal prop, final T value, final Accept
             callback.accepted(prop, value);
         }
     }
+
+    @Override
+    public void requestValue(Sync.Receiver<T> callback) {
+        callback.receive(mem.get());
```

Comment:
@stepanvalyavskiy we need to check if memory has any value before syncing it

Reply:
memory stores a value of type `T`. So it's only possible to check  that value not eq to `null` 
but we never pass nulls as arguments. Nullobjects instead.
Two ways:
* Change `Acceptor<T>` to `Acceptor<T extends Nullable>` and let `Decision` implement it. @override
* chech